mdetsims/sim_utils.py

```diff
@@ -30,8 +30,12 @@ class Sim(object):
 
     Parameters
     ----------
+    frac_ex : float, 0 to 1
+        The fraction of galxies with extra shear.
     rng : np.random.RandomState
         An RNG to use for drawing the objects.
+    sel : np.random.RandomState
```

Comment:
Let's rename this `sel` parameter and attribute to something like `sel_rng` or `rng_sel`.